### ISSUE TITLE
zsh, fish support

### ISSUE THREAD

**Issue by Altai-man** [7ojo/perl6-powerline-prompt#1]
Judging by https://github.com/banga/powerline-shell#zsh, it shouldn't be too hard to do, though it doesn't work correctly with a configuration from there now, I didn't investigate further yet.

Also, the performance is not at its best, though, I'm sure it will be fixed with time.

**Comment by 7ojo** [comment]
Yep about that performance. I have thoughts about doing a daemon example and client communication using plain unix commands, or make a thin client using some other language.
